### PR TITLE
Feature/cdf 9432 plot data extraction template

### PR DESCRIPTION
**Pull request by minshi-cognite**

### CODE REVIEW

File: advanced/Plot data extraction demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Plot data extraction demo\n",
+    "\n",
+    "This notebook demonstrate how to extract point data from plot using plot data extraction endpoint.\n",
```
Comment (sanderland) [inline advanced/Plot data extraction demo.ipynb:9]: demonstrates

File: advanced/Plot data extraction demo.ipynb [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+    "\n",
+    "Current version only support curve plots similar to the example which has no legend."
```
Comment (sanderland) [inline advanced/Plot data extraction demo.ipynb:11]: The currrent version only supports line plots without a legend, similar to the example.

File: advanced/Plot data extraction demo.ipynb [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+    "                       client_name=\"dshub\",\n",
+    "                      )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load an plot example image from google drive\n",
+    "Load an image has a plot, draw a boundingbox on the axes of the plot and copy the value (4 digit values sperated by comma) of the boundingbox to the cell below."
```
Comment (sanderland) [inline advanced/Plot data extraction demo.ipynb:61]: "Load an image which includes a plot, draw a bounding box coinciding with the axes of the plot and copy the value (4 numeric values separated by commas) of the bounding box to the cell below."

**Review by sanderland — APPROVED** [review]
Some small suggestions. It might be good to update this when we have SDK support

**Comment by minshi-cognite** [comment]
> Some small suggestions. It might be good to update this when we have SDK support

Sure, we will update the example of sdk usage after having sdk support

**Pull request merged**